src/main/java/com/quimify/api/health/HealthService.java

```diff
@@ -0,0 +1,61 @@
+package com.quimify.api.health;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.context.annotation.RequestScope;
+
+import com.quimify.api.classification.ClassificationService;
+import com.quimify.api.error.ErrorService;
+import com.quimify.api.inorganic.InorganicService;
+import com.quimify.api.molecularmass.MolecularMassService;
+import com.quimify.api.organic.OrganicService;
+
+@Service
+@RequestScope // For doing unique every Health Request and errors, then deleting everything
+class HealthService {
+
+    public final InorganicService inorganicService;
+    public final OrganicService organicService;
+    public final MolecularMassService molecularMassService;
+    public final ClassificationService classificatService;
+
+    @Autowired
+    ErrorService errorService;
+
+    public HealthService(InorganicService inorganicService, OrganicService organicService,
```

Comment:
Just use `@Autowired` like with `
@Autowired
ErrorService errorService;`.
There is no need for a constructor

Reply:
You're absolutely right.